### DISCUSSION TITLE
Welcome to winget-create Discussions!

### DISCUSSION THREAD

**Discussion by denelon** [microsoft/winget-create#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BAW2501** [comment]
how do i tell winget-create to silently install?

**Reply by denelon** [reply]
@BAW2501
are you referring to `winget install wingetcreate --silent`

or are you referring to the manifest generated by the tool?

**Reply by BAW2501** [reply]
Referring to the manifest file i how do i add command line args like \s for example to the installer?

**Reply by denelon** [reply]
@BAW2501 the wingetcreate tool will attempt to detect the installer type. If it is able to determine MSIX, MSI, NullSoft, Inno, or one of the other known installer types, the switches are not needed. If the installer type is .exe and the tool is not able to detect, you would likely need to look up the correct argument(s).

They live in the "Installer" node in the YAML manifest.

```YAML
InstallerSwitches:
    Custom: 
    Silent: 
    SilentWithProgress: 
```

**Reply by BAW2501** [reply]
Thank you this is exactly what i was looking for

**Comment by OfficialEsco** [comment]
ETA on next release?

**Reply by denelon** [reply]
~~No commitment here, but I'd expect:~~

~~DEV build likely in late June or early July.~~
~~Stable build likely in September.~~

Next business day or two.

**Reply by denelon** [reply]
Too many different repositories 🤦‍♂️

I thought I was replying in winget-cli.

**Reply by denelon** [reply]
@OfficialEsco
We ran into an issue with a dependency delaying the release. We're still working on it.
